InnerEye/Azure/azure_config.py

```diff
@@ -242,6 +246,67 @@ def fetch_run(self, run_recovery_id: str) -> Run:
         """
         return fetch_run(workspace=self.get_workspace(), run_recovery_id=run_recovery_id)
 
+    def get_or_create_dataset(self, azure_dataset_id: str) -> FileDataset:
+        """
+        Looks in the AzureML datastore for a dataset of the given name. If there is no such dataset, a dataset is
+        created and registered, assuming that the files are in a folder that has the same name as the dataset.
+        For example, if azure_dataset_id is 'foo', then the 'foo' dataset should be pointing to the folder
+        <container_root>/datasets/foo
+
+        WARNING: the behaviour of Dataset.File.from_files, used below, is idiosyncratic. For example,
```

Comment:
I don't think I understand this paragraph

Reply:
This is comment that David left here. I've removed it, and I think also fixed the problem he wishes to describe here.